### PR TITLE
Add deprecation message to dotnet migrate

### LINKED ISSUE #15510: Review `dotnet migrate` documentation
https://docs.microsoft.com/en-us/dotnet/core/tools/dotnet-migrate

### PR DESCRIPTION
**Pull request by wli3**
## Summary

Add deprecation message to dotnet migrate

Fixes https://github.com/dotnet/docs/issues/15510

### CODE REVIEW

File: docs/core/tools/dotnet-migrate.md [hunk 1]
```diff
@@ -20,6 +20,8 @@ dotnet migrate [-h|--help]
 
 ## Description
 
+This command is deprecated. It is no longer available in and after .NET Core 3.x SDK. It can only migrate a Preview 2 .NET Core project to a 1.x .NET Core project which is currently out of support.
```
Comment (mairaw) [inline docs/core/tools/dotnet-migrate.md]: This makes line 48 redundant. Should you remove that one then?

Comment (wli3) [reply docs/core/tools/dotnet-migrate.md]: I moved line 42 up there. And removed the next paragraph. Looks better?

**Review by mairaw — APPROVED** [review]
Thanks @wli3. Made a small change plus reviewed the date for us to merge this.

**Comment by mairaw** [comment]
We usually squash and merge docs PRs @wli3. Thanks!

**Pull request merged**